index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> <br>
+    <title>Odin</title>
+    
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <a href="chocopud.html">Chocolate Puddino</a> <br> <br>
```

Comment:
Create more recipes as per the instruction of the project.  Also, create a basic style sheet, where at least the :hover is distinguished from the links.  Use something like "text-decoration: none;" for the a tag and use a :hover style.  

The project requests not to style unecessarily, yes, but you must style enough for functionality to be user friendly.